app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    old_file = parts[1]
+    new_file = parts[2]
```

Comment:
Try to use unpacking here, but before check if the command has 3 parameters and if the command if equal to cp